integration_tests/e2e/overview/educationAndTrainingTab.cy.ts

```diff
@@ -38,7 +38,47 @@ context('Prisoner Overview page - Education And Training tab', () => {
       educationAndTrainingPage //
         .activeTabIs('Education and training')
         .hasFunctionalSkillsDisplayed()
-        .hasCompletedInPrisonQualificationsDisplayed()
+        .hasCompletedInPrisonQualificationsLast12MonthsDisplayed()
+    })
+
+    it('should display courses and qualification data if prisoner has completed courses or qualifications in last 12 months', () => {
+      // Given
+      cy.task('stubGetInductionShortQuestionSet')
+      cy.task('stubLearnerEducationWithCoursesQualificationsInLast12Months')
+
+      cy.signIn()
+      const prisonNumber = 'G6115VJ'
+      cy.visit(`/plan/${prisonNumber}/view/overview`)
+      const overviewPage = Page.verifyOnPage(OverviewPage)
+
+      // When
+      overviewPage.selectTab('Education and training')
+      const educationAndTrainingPage = Page.verifyOnPage(EducationAndTrainingPage)
+
+      // Then
+      educationAndTrainingPage //
+        .activeTabIs('Education and training')
+        .hasCompletedInPrisonQualificationsLast12MonthsDisplayed()
+        .hasCompletedInPrisonQualificationRecordDisplayed('GCSE Maths')
+    })
+
+    it('should display message if prisoner has no completed courses or qualifications in last 12 months', () => {
+      // Given
+      cy.task('stubGetInductionShortQuestionSet')
+
+      cy.signIn()
+      const prisonNumber = 'G6115VJ'
+      cy.visit(`/plan/${prisonNumber}/view/overview`)
+      const overviewPage = Page.verifyOnPage(OverviewPage)
+
+      // When
+      overviewPage.selectTab('Education and training')
+      const educationAndTrainingPage = Page.verifyOnPage(EducationAndTrainingPage)
+
+      // Then
+      educationAndTrainingPage //
+        .activeTabIs('Education and training')
+        .hasNoCoursesAndQualificationsLast12MonthsMessageDisplayed()
```

Comment:
Again, I don't follow this - where is the call to a stub in this test that says the prisoner has no completed courses in the last 12 months?
(also, "no completed courses in the last 12 months" is subtly different to "no courses at all" - do we have that covered?)